### PR TITLE
Lorenz 63 docs

### PR DESCRIPTION
**Pull request by AndreyAPopov**
This is still a work in progress

### CODE REVIEW

File: docs/references.bib [hunk 1]
```diff
@@ -63,3 +63,22 @@ @article{SLH70
...
+  number={2},
+  pages={130--141},
+  year={1963}
+}
+
+@book{Str18,
+  title={Nonlinear dynamics and chaos with student solutions manual: With applications to physics, biology, chemistry, and engineering},
+  author={Strogatz, Steven H},
+  year={2018},
+  publisher={CRC press}
```
Comment (Steven-Roberts) [inline docs/references.bib:83]: Include doi, eddition, and ISBN: [10.1201/9780429492563](https://doi.org/10.1201/9780429492563)

File: docs/references.bib [hunk 2]
```diff
@@ -63,3 +63,22 @@ @article{SLH70
...
+  author={Lorenz, Edward N},
+  journal={Journal of atmospheric sciences},
+  volume={20},
+  number={2},
+  pages={130--141},
+  year={1963}
+}
+
+@book{Str18,
+  title={Nonlinear dynamics and chaos with student solutions manual: With applications to physics, biology, chemistry, and engineering},
```
Comment (Steven-Roberts) [inline docs/references.bib:80]:
```suggestion
  title={Nonlinear Dynamics and Chaos: {W}ith applications to physics, biology, chemistry, and engineering},
```

File: docs/references.bib [hunk 3]
```diff
@@ -63,3 +63,22 @@ @article{SLH70
...
+
+
+@article{Lor63,
+  title={Deterministic nonperiodic flow},
+  author={Lorenz, Edward N},
+  journal={Journal of atmospheric sciences},
+  volume={20},
+  number={2},
+  pages={130--141},
+  year={1963}
```
Comment (Steven-Roberts) [inline docs/references.bib:76]: Include doi: [https://doi.org/10.1175/1520-0469(1963)020<0130:DNF>2.0.CO;2](https://doi.org/10.1175/1520-0469(1963)020%3C0130:DNF%3E2.0.CO;2)

File: toolbox/+otp/+lorenz63/Lorenz63Problem.m [hunk 4]
```diff
@@ -1,23 +1,44 @@
...
+    % Notes
+    % -----
+    % +---------------------+-----------------------------------------+
+    % | Type                | ODE                                     |
+    % +---------------------+-----------------------------------------+
+    % | Number of Variables | 3                                       |
+    % +---------------------+-----------------------------------------+
+    % | Stiff               | no                                      |
+    % +---------------------+-----------------------------------------+
+    % | Chaotic             | typically, depending on parameters      |
```
Comment (Steven-Roberts) [inline toolbox/+otp/+lorenz63/Lorenz63Problem.m:30]: Let's leave chaotic out of the table notes. We can mention it elsewhere.

File: toolbox/+otp/+lorenz63/Lorenz63Problem.m [hunk 5]
```diff
@@ -1,23 +1,44 @@
...
-    %  Lorenz, Edward N. "Deterministic nonperiodic flow."
-    %  Journal of the atmospheric sciences 20, no. 2 (1963): 130-141.
+    % Notes
+    % -----
+    % +---------------------+-----------------------------------------+
+    % | Type                | ODE                                     |
+    % +---------------------+-----------------------------------------+
+    % | Number of Variables | 3                                       |
+    % +---------------------+-----------------------------------------+
+    % | Stiff               | no                                      |
```
Comment (Steven-Roberts) [inline toolbox/+otp/+lorenz63/Lorenz63Problem.m:28]: not typically, depending on $\sigma$, $\rho$, and $\beta$.

**Review by Steven-Roberts — CHANGES_REQUESTED** [review]
Some edits on first review. I still need to view it locally for final review

File: toolbox/+otp/+lorenz63/Lorenz63Problem.m [hunk 6]
```diff
@@ -1,26 +1,58 @@
...
-    %  Lorenz, Edward N. "Deterministic nonperiodic flow."
-    %  Journal of the atmospheric sciences 20, no. 2 (1963): 130-141.
-    %
-    % For a more detailed description also take a look at
-    %
-    %  Strogatz, Steven H. Nonlinear dynamics and chaos: with applications to
-    %  physics, biology, chemistry, and engineering. Westview press, 2014.
+    % See also
+    % --------
+    % :doc:`Lorenz '96 Problem </problems/Lorenz96Problem>`
```
Comment (elswit) [inline toolbox/+otp/+lorenz63/Lorenz63Problem.m:37]: @elswit This is useful

**Pull request merged**